### DISCUSSION TITLE
Can the "organization belongs to" be changed after the organization is created?

### DISCUSSION THREAD

**Discussion by neusoft** [community/community#101537] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello!

I encountered some issues when I was transforming the user to an organization and would like to ask for some advice.

The questions are:

1. Can the "organization belongs to" filled in Step 1 be changed after the organization is created?
2. What are the steps to modify it? 
3. What purpose does this information serve?

![image](https://github.com/community/community/assets/156869505/3cb2f174-8489-444b-9738-63c48bdb072a)

**Comment by n31t** [comment, marked as answer]
1)Can the "organization belongs to" filled in Step 1 be changed after the organization is created?
Usually, the "organization belongs to" information, once set during the organization creation process, cannot be directly changed. It is a fundamental piece of information that establishes the relationship or hierarchy between organizations, particularly if there is a parent organization.

2)If there's a need to modify the "organization belongs to" information, the common approach is to create a new organization with the correct parent or belonging information and then migrate the necessary repositories, teams, and settings to the new organization. 
1.Create a New Organization:
Create a new organization with the correct parent or belonging information.
2.Migrate Repositories:
Transfer or fork repositories from the old organization to the new one.
3.Migrate Teams and Settings:
Recreate teams and settings in the new organization to match the old one.
4.Update References:
Update any references, webhooks, or integrations to point to the new organization.

3)What purpose does this information serve?
The "organization belongs to" information helps establish a hierarchical structure among organizations, especially in scenarios where there are parent and child organizations. It might impact permissions, access controls, and administrative relationships between organizations. Understanding the organizational structure can be crucial for managing collaboration and access within larger projects or entities.

**Reply by neusoft** [reply]
Thank you very much!
